data/data/bootstrap/files/usr/local/bin/bootstrap-pivot.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euo pipefail
+# Update bootstrap node with latest content. Required for OKD as it starts with FCOS image
+# and needs to be updated to the machine-os-content used by masters / nodes.
+
```

Comment:
Please source `/usr/local/bin/bootstrap-service-record.sh` to record the progress of this service. Add calls to `record_service_stage_start` and `record_service_stage_success` also if there are meaningful stage for this service.

See https://github.com/openshift/installer/pull/4742

Reply:
Good idea, done